.github/workflows/cabal.yml

```diff
@@ -66,6 +75,8 @@ jobs:
       run: cabal install bech32 --overwrite-policy=always --install-method=copy --installdir=bin
 
     - name: 🧪 Test
+      # Skipping test for ghc-8.10.3 because it currently fails
```

Comment:
Well, why does it fail? :upside_down_face:

Reply:
Template Haskell issues.  See: https://github.com/input-output-hk/bech32/pull/29

Hoping someone can fix that up.